homeassistant/components/isy994/sensor.py

```diff
@@ -264,32 +260,3 @@ def name(self) -> str:
         base_name = self._name or str(self._node.name)
         name = COMMAND_FRIENDLY_NAME.get(self._control, self._control)
         return f"{base_name} {name.replace('_', ' ').title()}"
-
-
```

Comment:
We should leave the sensor entities in place for 3-4 months to allow users some time to convert over so they don't feel like they can't upgrade right away

Reply:
Reverted.